src/test/kotlin/no/nav/syfo/testhelper/ExternalMockEnvironment.kt

```diff
@@ -20,18 +20,18 @@ class ExternalMockEnvironment private constructor() {
 
     var environment = testEnvironment()
     val mockHttpClient = mockHttpClient(environment = environment)
-    private val redisConfig = environment.redisConfig
-    val redisCache = RedisStore(
+    private val redisConfig = environment.valkeyConfig
+    val redisCache = ValkeyStore(
```

Comment:
`cache` eller `valkeyStore`? 
```suggestion
    val cache = ValkeyStore(
```

Reply:
Vi bruker faktisk en test-redis her (embedded-redis), så tenker det er greit at det fortsatt er reflektert i navnet..

Reply:
Aha 💡